### PR TITLE
Updated references and import paths to the new repository location

### PR DESCRIPTION
**Pull request by omeryahud**
Signed-off-by: Omer Yahud <oyahud@redhat.com>

### CODE REVIEW

**Comment by omeryahud** [comment]
@ksimon1 @akrejcir Can you take a look please?

File: build/generate.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 ./vendor/k8s.io/code-generator/generate-groups.sh \
 	all \
-	github.com/MarSik/kubevirt-ssp-operator/pkg/client \
-	github.com/MarSik/kubevirt-ssp-operator/pkg/apis \
+	github.com/kubevirt/kubevirt-ssp-operator/pkg/client \
```
Comment (akrejcir) [inline build/generate.sh:4]: Can we change the package path to `kubevirt.io/kubevirt-ssp-operator/...`, like other kubevirt components have?

Comment (akrejcir) [reply build/generate.sh:4]: I see that some components use the github path. So this is probably ok.

Comment (omeryahud) [reply build/generate.sh:4]: Yea, I noticed all other components except for kubevirt are prefixed with `github.com` and not `kubevirt.io`, maybe we can change that in the future

**Pull request merged**